Makefile

```
@@ -1,4 +1,4 @@
-VERSION = 0.11
+VERSION = 1.0-2
```

```suggestion
VERSION = 1.0
```
Just 1.0. The "release" field of the RPM is not considered part of the software version.